cni/pkg/cmd/root.go

```diff
@@ -132,25 +132,25 @@ var rootCmd = &cobra.Command{
 			// if it is, we do NOT remove the plugin, and do
 			// NOT do ambient watch server cleanup
 			defer func() {
-				var isUpgrade bool
+				var shouldStopCleanup bool
```

Comment:
not: the polarity changing here makes it difficult for me to know if the logic switch is intentional or not. Before the check was "is this an upgrade" now it's "should we stop a thing". Ideally, the polarity could stay the same